.travis.yml

```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 
 sbt_args: -no-colors
-script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc
```

Comment:
No idea, I set this up 3 years ago to have cross-version builds. Feel free to change it as you want.